src/math_ops.jl

```diff
@@ -70,7 +70,7 @@ julia> det(A)
     if dim == 1
         return quote
             $(Expr(:meta, :inline))
-            get_data(t)[$idx(1,1)]
+            get_data(t)[$(idx(1,1))]
```

Comment:
nice brackets... I interpolated the function into the expression and not the index